components/datatypes/internal-search.schema.json

```diff
@@ -0,0 +1,121 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/internal-search",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Internal Search",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Used for internal site searches and includes all related search behaviors and details.",
+  "definitions": {
+    "internalSearch": {
+      "properties": {
+        "xdm:autocompleteSearch": {
+          "title": "Search Autocomplete Value",
+          "type": "string",
+          "description": "Tracks when a visitor uses a suggested/autocompleted search value to execute the search"
+        },
+        "xdm:numberofSearchResults": {
+          "title": "Number of Results",
+          "type": "integer",
+          "description": "Count of the total number of search results returned."
+        },
+        "xdm:productFindingMethods": {
+          "title": "Search Product Finding Method",
+          "type": "string",
+          "description": "The internal search term value with merchandising binding. Will show what term was searched for immediately before viewing a product."
+        },
+        "xdm:refineSearch": {
+          "title": "Refinement Value",
+          "type": "string",
+          "description": "Number of times an internal search refinement has occurred."
+        },
+        "xdm:refineType": {
+          "title": "Search Refinement Type",
+          "type": "array",
+          "description": "Captures the refinement type(s) applied to the search results (i.e. department, brand, price, in-store, review rating, color, material, etc.)"
+        },
+        "xdm:resultsPageNumber": {
+          "title": "Search Results Page Number",
+          "type": "integer",
+          "description": "For paginated search results, tracks the page of results the visitor is viewing."
+        },
+        "xdm:search": {

```

Comment:
Can we rename this to something more meaningful ? "searchOccurences" ?

Reply:
Yes, I can change this.